google/cloud/internal/backoff_policy.cc

```diff
@@ -42,7 +42,7 @@ std::chrono::milliseconds ExponentialBackoffPolicy::OnCompletion() {
     generator_ = google::cloud::internal::MakeDefaultPRNG();
   }
   std::uniform_int_distribution<microseconds::rep> rng_distribution(
-      current_delay_range_.count() / 2, current_delay_range_.count());
+      (std::max)(initial_delay_.count(), current_delay_range_.count() / 2), current_delay_range_.count());
```

Comment:
Could we instead also take care of this during initialization by ensuring "current / 2 >= initial"?

Reply:
Done